src/main/java/com/iridium/iridiumskyblock/listeners/PlayerInteractListener.java

```diff
@@ -21,12 +21,24 @@
 
 public class PlayerInteractListener implements Listener {
 
-    @EventHandler(ignoreCancelled = true)
+    @EventHandler
```

Comment:
oh! funny story! because spigot will automatically cancel the interact event if you're not clicking on a block! which means you could only deposit the crystal when looking at the ground :)

Reply:
Sounds like a spigot thing to do